src/main/java/com/alkemy/ong/exception/NewsNotFoundException.java

```diff
@@ -0,0 +1,10 @@
+package com.alkemy.ong.exception;
+
+import javassist.NotFoundException;
+
+public class NewsNotFoundException extends NotFoundException {
```

Comment:
No me di cuenta de esto, pero deberíamos cambiar esta `NotFoundException` por `RuntimeException`. Si podes cambialo en las otras not found exception 🙏🏼